lapack/testlapack/dlantb_bench.go

```diff
@@ -0,0 +1,54 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testlapack
+
+import (
+	"fmt"
+	"testing"
+
+	"golang.org/x/exp/rand"
+
+	"gonum.org/v1/gonum/blas"
+	"gonum.org/v1/gonum/lapack"
+)
+
+var result float64
```

Comment:
These are irritating. It would be nice to have a clear explanation from the Go team about this that is included in vetting tools.